test/integration/component/test_persistent_networks.py

```diff
@@ -230,6 +231,116 @@ def tearDown(self):
             raise Exception("Warning: Exception during cleanup : %s" % e)
         return
 
+    '''
+    Verifies creation of bridge on KVM host
+    '''
+    def verify_bridge_creation(self, host, vlan_id):
+        username = self.hostConfig["username"]
+        password = self.hostConfig["password"]
+        try:
+            ssh_client = self.get_ssh_client(host.ipaddress, username, password)
+            res = ssh_client.execute("ip addr | grep breth0-" + str(vlan_id) + " > /dev/null 2>&1; echo $?")
```

Comment:
Minor note: in my tests the bridge name was not this, but `breth1-<VLAN>`

Reply:
yes, would be good to get the guest physical network name cloudbrx  then physical nic ethx